DOL.WHD.Section14c.Web/src/v4/dol-footer.component.ts

```diff
@@ -10,7 +10,7 @@ import { Component } from '@angular/core';
           <img
             class="brand--img"
             src="images/dol-logo-white.png"
-            alt="DOL Logo"
+            alt="Seal of the United States Department of Labor"
```

Comment:
This was one of the first lessons I learned about proper accessibility: never alt an image with the word "logo". Lol